### ISSUE TITLE
CI Failure: HTTP [429] Failed: Network error: Too Many Requests

### ISSUE THREAD

**Issue by pjanotti** [open-telemetry/opentelemetry-dotnet-instrumentation#1642]
Error running CI for PR:

- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/actions/runs/3516337687
- https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/actions/runs/3516337687/jobs/5892769153

**Comment by pjanotti** [comment]
@Kielek I think you commented or took a quick look at this. At first, it looks like we have to throttle, somehow, our HTTP requests. Do you know if that is really the case?

**Comment by Kielek** [comment]
Yes, it is throttling. The workaround is to use GITHUB_TOKEN. PR is ready.
